tests/components/zwave_js/conftest.py

```diff
@@ -82,6 +82,12 @@ def nortek_thermostat_state_fixture():
     return json.loads(load_fixture("zwave_js/nortek_thermostat_state.json"))
 
 
+@pytest.fixture(name="window_cover_state", scope="session")
```

Comment:
@MartinHjelmare should I adjust the name here to `chain_actuator_zws12_state`?

Reply:
Yes, please.